pkg/meta/store.go

```diff
@@ -47,9 +48,21 @@ type Store struct {
 
 // NewStore creates a backend storage.
 func NewStore(cfg Config) (*Store, error) {
+	if len(cfg.Buckets) < 1 {
+		return nil, fmt.Errorf("config bucket can not be nil")
```

Comment:
s/nil/empty

`empty` is a user's perspective and nil is only a coder's perspective.